u-root.go

```diff
@@ -32,6 +32,10 @@ var (
 	build, format, tmpDir, base, outputPath *string
 	useExistingInit                         *bool
 	extraFiles                              multiFlag
+	templates                               = map[string][]string{
+		// This is just a test, please tell me what you'd like.
```

Comment:
I think we should have at least a "core" set of commands, like ps, ls, cat and so on - the ones you can't live without. Then an "all" template. Other templates can be added later. If you agree I can actually add a comment with a list of what I consider "core" and we can add or remove from that

Reply:
go for it. Thanks!

Reply:
consider adding "compact" later.

Reply:
sounds good, but in my experiments, adding commands in either source or bb mode doesn't change much the ramfs size (or I tested with the wrong commands). If the goal of "compact" is to significantly reduce size, this may be not necessary, otherwise let's go for it :)

Reply:
something like this?

```
			"cmds/ansi",
			"cmds/builtin",
			"cmds/cbmem",
			"cmds/chroot",
			"cmds/cp",
			"cmds/date",
			"cmds/df",
			"cmds/dirname",
			"cmds/echo",
			"cmds/ed",
			"cmds/field",
			"cmds/fmap",
			"cmds/free",
			"cmds/gpgv",
			"cmds/grep",
			"cmds/hexdump",
			"cmds/id",
			"cmds/insmod",
			"cmds/io",
			"cmds/kexec",
			"cmds/lddfiles",
			"cmds/losetup",
			"cmds/lsmod",
			"cmds/mkfifo",
			"cmds/mount",
			"cmds/mv",
			"cmds/ntpdate",
			"cmds/ping",
			"cmds/ps",
			"cmds/pxeboot",
			"cmds/rm",
			"cmds/rsdp",
			"cmds/rush",
			"cmds/shutdown",
			"cmds/sort",
			"cmds/stty",
			"cmds/sync",
			"cmds/true",
			"cmds/umount",
			"cmds/uniq",
			"cmds/validate",
			"cmds/wc",
			"cmds/which",
			"cmds/archive",
			"cmds/boot",
			"cmds/cat",
			"cmds/chmod",
			"cmds/cmp",
			"cmds/console",
			"cmds/cpio",
			"cmds/dd",
			"cmds/dhclient",
			"cmds/dmesg",
			"cmds/false",
			"cmds/find",
			"cmds/freq",
			"cmds/gpt",
			"cmds/gzip",
			"cmds/hostname",
			"cmds/init",
			"cmds/installcommand",
			"cmds/ip",
			"cmds/kill",
			"cmds/ln",
			"cmds/ls",
			"cmds/mkdir",
			"cmds/mknod",
			"cmds/modprobe",
			"cmds/msr",
			"cmds/netcat",
			"cmds/pci",
			"cmds/pflask",
			"cmds/printenv",
			"cmds/pwd",
			"cmds/readlink",
			"cmds/rmmod",
			"cmds/run",
			"cmds/seq",
			"cmds/sleep",
			"cmds/switch_root",
			"cmds/tail",
			"cmds/tee",
			"cmds/truncate",
			"cmds/uname",
			"cmds/unshare",
			"cmds/vboot",
			"cmds/wget",
```

Reply:
and the name of this template would be "core"

Reply:
we will also have to add (in another PR maybe?) builds for each template in the tests to make sure that they are kept up-to-date (e.g. if a command goes away we will catch it).